lib/internal/http2/compat.js

```diff
@@ -40,12 +40,18 @@ let statusMessageWarned = false;
 // close as possible to the current require('http') API
 
 function assertValidHeader(name, value) {
-  if (name === '' || typeof name !== 'string')
-    throw new errors.TypeError('ERR_INVALID_HTTP_TOKEN', 'Header name', name);
-  if (isPseudoHeader(name))
-    throw new errors.Error('ERR_HTTP2_PSEUDOHEADER_NOT_ALLOWED');
-  if (value === undefined || value === null)
-    throw new errors.TypeError('ERR_HTTP2_INVALID_HEADER_VALUE');
+  var err;
```

Comment:
Nit: can we use `let`? (I don't think there's many/any vars in http2 except for `for` loops.)

Reply:
I am still under the impression that we favor `var` in cases like this...might be just outdated impressions, I'll fix that.